utoipa-gen/src/schema_type.rs

```diff
@@ -372,26 +400,36 @@ pub enum Variant {
     Password,
     #[cfg(feature = "uuid")]
     Uuid,
+    #[cfg(feature = "ulid")]
+    Ulid,
     Custom(String),
 }
 
 impl Parse for Variant {
     fn parse(input: syn::parse::ParseStream) -> syn::Result<Self> {
-        const FORMATS: [&str; 10] = [
+        const FORMATS: [&str; 11] = [
             "Int32", "Int64", "Float", "Double", "Byte", "Binary", "Date", "DateTime", "Password",
-            "Uuid",
+            "Uuid", "Ulid",
         ];
         let known_formats = FORMATS
             .into_iter()
             .filter(|_format| {
-                #[cfg(feature = "uuid")]
+                #[cfg(all(feature = "uuid", feature = "ulid"))]
                 {
                     true
                 }
-                #[cfg(not(feature = "uuid"))]
+                #[cfg(all(not(feature = "uuid"), feature = "ulid"))]
                 {
                     _format != &"Uuid"
                 }
+                #[cfg(all(feature = "uuid", not(feature = "ulid")))]
+                {
+                    _format != &"Ulid"
+                }
+                #[cfg(all(not(feature = "uuid"), not(feature = "ulid")))]
+                {
+                    _format != &"Ulid" && _format != &"Ulid"
+                }
             })
```

Comment:
This seems a bit fishy

Reply:
Can we have a better code for this? I cannot come up with it.

Reply:
Perhaps we can have this as is, and change it in future.